build-libssp.sh

```diff
@@ -80,7 +80,7 @@ for arch in $ARCHS; do
     [ -z "$CLEAN" ] || rm -rf build-$arch
     mkdir -p build-$arch
     cd build-$arch
-    $MAKE -f ../Makefile -j$CORES CROSS=$arch-w64-mingw32-
+    CC=$arch-w64-mingw32-clang $MAKE -f ../Makefile -j$CORES CROSS=$arch-w64-mingw32-
```

Comment:
FWIW for this one, I would think that the makefile (a handwritten one) wouldn't pick up CC from the environment unless it's forced on make like `make ... CC=mycompiler`.

Reply:
Yup, setting a bogus CC when running `build-libssp.sh` doesn't have any effect right now, so this change isn't needed.

Reply:
Alright I'll remove that, and also replace everything with `unset CC`